diff --git a/Android.bp b/Android.bp
index 235586c..1595d5e 100644
--- a/Android.bp
+++ b/Android.bp
@@ -53,7 +53,7 @@ rust_library {
         "usage",
     ],
     rustlibs: [
-        "libbitflags",
+        "libbitflags-1.3.2",
         "libclap_lex",
         "libonce_cell",
     ],
@@ -82,7 +82,7 @@ rust_binary {
         "usage",
     ],
     rustlibs: [
-        "libbitflags",
+        "libbitflags-1.3.2",
         "libclap",
         "libclap_lex",
         "libonce_cell",
